kolibri/plugins/learn/assets/src/views/content-card-group-carousel/index.vue

```diff
@@ -278,6 +277,7 @@
       overflow-y: visible
 
     &-card
+      left: 0
```

Comment:
Sets default alignment of cards. Without this, first card flies across carousel to its place when RTL. I think it has something to do with it being first in the v-for and the dynamic `style` attribute not being ready in time. So it gets rendered to `left: 0` by default before the style tag is applied and the shift animates it. Putting this here sets it to `right: 0` in RTL